package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "fh-wfm-file-storage",
+  "version": "0.1.0",
+  "description": "A WFM module providing file storage support",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/feedhenry-raincatcher/raincatcher-file-storage.git"
+  },
+  "scripts": {
+    "test": "grunt test"
+  },
+  "keywords": [
+    "wfm",
+    "file"
+  ],
+  "author": "feedhenry-raincatcher@redhat.com",
+  "license": "MIT",
+  "dependencies": {
+    "debug": "^2.6.3",
+    "fh-wfm-mediator": "0.3.1",
```

Comment:
`fh-wfm-mediator` is a devDependency as the mediator is passed from the consuming application.